pyface/qt/QtWebKit.py

```diff
@@ -28,6 +28,38 @@
         from PyQt5.QtWebKit import *
         from PyQt5.QtWebKitWidgets import *
 
+elif qt_api == "pyqt6":
+    from PyQt5.QtWidgets import *
+
+    try:
+        from PyQt6.QtWebEngine import *
+        from PyQt6.QtWebEngineWidgets import (
+            QWebEngineHistory as QWebHistory,
+            QWebEngineHistoryItem as QWebHistoryItem,
+            QWebEnginePage as QWebPage,
+            QWebEngineView as QWebView,
+            QWebEngineSettings as QWebSettings,
+        )
+    except ImportError:
+        from PyQt6.QtWebKit import *
+        from PyQt6.QtWebKitWidgets import *
+
+elif qt_api == "pyside6":
+    from PySide6.QtWidgets import *
+
+    # WebKit is currently in flux in PySide6
+    try:
+        from PySide6.QtWebEngineWidgets import (
```

Comment:
I think some of these names need to be imported from`QtWebEngineCore` rather than `QtWebEngineWidgets`. Testing with macOS 11.6.1 + Python 3.10 + PySide6 from PyPI, the following works for me:
```python
>>> from PySide6.QtWebEngineCore import QWebEngineSettings, QWebEnginePage, QWebEngineHistoryItem
>>> from PySide6.QtWebEngineWidgets import QWebEngineView
```

I'd guess the same changes need to be made for PyQt6, but I haven't tested.

Reply:
> I'd guess the same changes need to be made for PyQt6, but I haven't tested.

Confirmed: the following works for me with Python 3.10 + PyQt6 + PyQt6-WebEngine from PyPI:
```python
>>> from PyQt6.QtWebEngineCore import QWebEngineSettings, QWebEnginePage, QWebEngineHistoryItem
>>> from PyQt6.QtWebEngineWidgets import QWebEngineView
```